lib/lm_docstache/version.rb

```diff
@@ -1,3 +1,3 @@
 module LMDocstache
-  VERSION = "2.1.2"
```

Comment:
We don't need to change the major version...do we? Are we having breaking changes?

Reply:
Yes, it's a breaking change. We replaced Array to Hash in the options and might break if someone updates to minor version.